containers/podman-desktop
# feat: add rosetta support

feat: add rosetta support

### What does this PR do?

* Adds rosetta support to the configuration page of Podman.
* When disabling it will add the rosetta=false key
* When enabling, it will remove the rosetta key.

### Screenshot / video of UI




https://github.com/containers/podman-desktop/assets/6422176/e01e2d54-3f68-462d-8f38-0e5b29eb2c68



### What issues does this PR fix or reference?



Closes https://github.com/containers/podman-desktop/issues/7367

### How to test this PR?




1. Be on Mac
2. Disable the setting (set to false)
3. See it edited containers.conf and added rosetta=false
4. Switch to enable
5. See it removed the rosetta key.

Signed-off-by: Charlie Drage <charlie@charliedrage.com>
